src/hrflow_connectors/connectors/poleemploi/warehouse.py

```diff
@@ -0,0 +1,202 @@
+    sport = "L14"  # L14 -> Sport
+    achats = "M"  # M -> Achats / Comptabilité / Gestion
+    direction = "M13"  # M13 -> Direction d’entreprise
+    conseil = "M14"  # M14 -> Conseil/Etudes
+    ressources = "M15"  # M15 -> Ressources Humaines
+    secretariat = "M16"  # M16 -> Secrétariat/Assistanat
+    marketing = "M17"  # M17 -> Marketing /Stratégie commerciale
+    informatique = "M18"  # M18 -> Informatique / Télécommunication
+    transport = "N"  # N -> Transport / Logistique
+
+
+class PublieeDepuis(IntEnum):
+    # Valeurs possibles : 1, 3, 7, 14, 31
+    a = 1
+    b = 3
+    c = 7
+    d = 14
+    e = 31
+
+
+class ModeSelectionPartenaires(str, Enum):
+    inclus = "INCLUS"
+    exclu = "EXCLU"
+
+
+class DureeHebdo(str, Enum):
+    nonPrecise = "0"
+    tempsPlein = "1"
+    tempsPartiel = "2"
+
+
+class ReadJobsParameters(BaseModel):
+    access_token: str = Field(
+        ..., description="Bearer Token used to access Pole Emploi's API", repr=False
+    )
+    range: t.Optional[str]
+    sort: t.Optional[int]
+    # domaine : t.Optional[Domaine]
+    # codeROME : t.Optional[codeROME]
+    # theme : t.Optional[Theme]
+    # appellation : t.Optional[Appellation]
+    # secteurActivite : t.Optional[SecteurActivite]
+    experience: t.Optional[Experience]
+    # typeContrat : t.Optional[TypeContrat]
+    # natureContrat : t.Optional[NatureContrat]
+    origineOffre: t.Optional[OrigineOffreTag]
+    qualification: t.Optional[Qualification]
+    tempsPlein: t.Optional[bool]
+    # commune : t.Optional[Commune]
+    distance = (
+        10  # distance=0 pour pour obtenir seulement les offres d'une commune spécifique
+    )
+    departement: t.Optional[str]
+    inclureLimitrophes: t.Optional[bool]
+    # region : t.Optional[Region]
+    # paysContinent : t.Optional[PaysContinent]
+    # niveauFormation : t.Optional[NiveauFormation]
+    # permis : t.Optional[Permis]
+    motsCles: str
+    salaireMin: t.Optional[float]
+    periodeSalaire: t.Optional[
+        PeriodeSalaire
+    ]  # TODO: link salaireMin w/ periodeSalaire
+    accesTravailleurHandicape: t.Optional[bool]
+    offresMRS: t.Optional[bool]
+    grandDomaine: t.Optional[GrandDomaine]
+    experienceExigence: t.Optional[ExperienceExige]
+    publieeDepuis: t.Optional[PublieeDepuis]
+    minCreationDate: t.Optional[str]
+    maxCreationDate: t.Optional[str]
+
+    partenaires: t.Optional[
+        str
+    ]  # Il est possible de saisir plusieurs codes (séparateur ",").
+    modeSelectionPartenaires: t.Optional[ModeSelectionPartenaires]
+    dureeHebdo: t.Optional[DureeHebdo]
+    dureeHebdoMin: t.Optional[int]  # TODO : format HHMM
+    dureeHebdoMax: t.Optional[int]  # TODO : format HHMM
+    dureeContratMin: t.Optional[
+        float
+    ]  # TODO : Décimal positif (Séparateur de décimales: '.')
+    dureeContratMax: t.Optional[
+        float
+    ]  # TODO : Décimal positif (Séparateur de décimales: '.')
+    offresManqueCandidats: t.Optional[bool]
```

Comment:
@nedhirouebnou What is this TODO ? 
Why don't you do it ? 

Use TODO tree extension go over all your code and fix all them TODOs